test/NuGet.Tasks.Tests/PackageLineupPolicyTests.cs

```diff
@@ -22,8 +22,18 @@
 
 namespace NuGet.Tasks.Tests
 {
-    public class PackageLineupPolicyTests
+    public class PackageLineupPolicyTests : IDisposable
```

Comment:
Yup. xunit will create an instance of the test class for each test method and call .Dispose to cleanup after, regardless of test outcome. The same is true of test fixtures and test collections.